test/integration/src/test/java/io/pravega/test/integration/ControllerMetricsTest.java

```diff
@@ -207,6 +210,23 @@ public void streamMetricsTest() {
         checkStatsRegisteredValues(iterations * iterations, UPDATE_STREAM_LATENCY, TRUNCATE_STREAM_LATENCY);
     }
 
+    /**
+     * This test verifies that the Controller increments the metric for Zookeeper session expiration events correctly.
+     *
+     * @throws Exception
+     */
+    @Test(timeout = 60000)
```

Comment:
! minute is a long wait. Can we have a shortet timeout?

Reply:
Done.